contracts/contracts/eosio.system/src/voting.cpp

```diff
@@ -378,4 +379,8 @@ namespace eosiosystem {
       );
    }
 
+   bool voter_info::bp_status_active() const {
+         return current_time_point() - last_reassertion_time < microseconds( voter_info::reassertion_period * useconds_per_day );
```

Comment:
Use parentheses
return (current_time_point() - last_reassertion_time) < microseconds( voter_info::reassertion_period * useconds_per_day );

https://wiki.sei.cmu.edu/confluence/display/c/EXP00-C.+Use+parentheses+for+precedence+of+operation